home.admin/_cache.sh

```diff
@@ -429,12 +431,14 @@ elif [ "$1" = "valid" ]; then
 
   position=0
   lasttouch_overall=""
+  # shellcheck disable=SC2068
   for keystr in $@
   do
 
     # skip first parameter from script - thats the action string
     ((position++))
     if [ $position -eq 1 ]; then
+      # shellcheck disable=SC2145
       echo "# _cache.sh $@"
```

Comment:
Why not
```suggestion
       echo "# _cache.sh $*"
```
In this case?

Reply:
aiming to keep difficult to test changes to the minimum